### PR TITLE
Combine success and error messages together

### PR DESCRIPTION
**Pull request by karelhala**
We should show only one message when sabing data, so we gotta combine multiple success messages together. Same for errors, if there are any.

If there are only success messages show single success message, if there are only errors show single error message, if there are some errors and some successes show 2 messages with list of apps (which failed and which succeeded).

Screenshot from 2020-04-15 14-00-55

Screenshot from 2020-04-15 22-33-34

Screenshot from 2020-04-15 15-49-15

#### TODO
* [x] Add tests
* [x] Add pictures

### CODE REVIEW

**Comment by karelhala** [comment]
ping @mmenestr

File: src/Utilities/functions.js [hunk 1]
```diff
@@ -54,3 +55,59 @@ export const getSection = (key, schema = {}, storeSchema, success = () => {}) =>
         };
     }
 };
+
+export const concatApps = (apps = []) => {
+    return apps.reduce((acc, title, currentIndex) => (
+        `${acc}${title}${
+            currentIndex < apps.length - 1 ?
+                currentIndex < apps.length - 2 ? ',' : ' and'
```
Comment (Hyperkid123) [inline src/Utilities/functions.js:63]: is there any possibility of using `join()` on the main array?

**Comment by mmenestr** [comment]
Is it possible for the middle two to **just** say
"Email preferences for insights successfully saved" in green 
"Email preferences for  Policies unsuccessfully saved" in red.
Like change the green and red text to say that^ and get rid of the black sub text for both. Just to keep it more concise and clear.

File: src/Utilities/functions.js (hunk 1 above)
Comment (karelhala) [reply src/Utilities/functions.js:63]: Yeah, this might be a bit faster than the reduce in here.

Comment (karelhala) [reply src/Utilities/functions.js:63]: I had to change it to take empty and array with 1 element into account.

**Comment by karelhala** [comment]
@mmenestr sure thing!

**Pull request merged**